src/bindings/python/flux/uri/resolvers/lsf.py

```diff
@@ -0,0 +1,102 @@
+###############################################################
+# Copyright 2022 Lawrence Livermore National Security, LLC
+# (c.f. AUTHORS, NOTICE.LLNS, COPYING)
+#
+# This file is part of the Flux resource manager framework.
+# For details, see https://github.com/flux-framework.
+#
+# SPDX-License-Identifier: LGPL-3.0
+###############################################################
+
+import re
+import subprocess
+from pathlib import PurePath
+
+from flux.uri import FluxURIResolver, URIResolverPlugin
+
+
+"""
+This module supports the Flux URI resolver for IBM Spectrum LSF.
+
+It can be used like:
+  `flux uri --remote lsf:<JOBID>`.
+
+"""
```

Comment:
Module docstrings go at the top of the file, before any code (including imports).